Blazor_Foundation_6_Test_Shared/Documentation/Containers/Off-Canvas.razor

```diff
@@ -1,6 +1,419 @@
+        <Button DataToggle="offCanvasLeftOverlap">Open Right with Overlap</Button>
+        <Button DataToggle="offCanvasRightPush">Open Right with Push</Button>
+        <OpenCodeDev.Blazor.Foundation.Components.Containers.OffCanvas Position="position-left" Id="offCanvasLeftOverlap" DataTransition="overlap">
+            Your menu or Off-canvas content goes here 
+            <p>
+                Your menu or Off-canvas content goes here
+            </p>
+        </OpenCodeDev.Blazor.Foundation.Components.Containers.OffCanvas>
+        <OpenCodeDev.Blazor.Foundation.Components.Containers.OffCanvas Position="position-right" Id="offCanvasRightPush" DataTransition="push">
+            Your menu or Off-canvas content goes here
+            <p>
+                Your menu or Off-canvas content goes here
+            </p>
+        </OpenCodeDev.Blazor.Foundation.Components.Containers.OffCanvas>
+        <hr />
+    </SectionLeft>
+</SectionWrapper>
+<SectionWrapper>
+    <SectionLeft>
+        <h2>Combining with Title Bar</h2>
+        <p>
+            If you need a simple bar to contain your hamburger icon/s and toggle the off-canvas, <Code Content="PFN0aWNreVRpdGxlQmFyPg=="></Code> is here to help. It supports left- and right-aligned sections. You can add your off-canvas toggle triggers here:
+        </p>
+        <HighlightCS Language="html" Full="true" HeaderIcon="mdi-language-html5" HeaderBGcolor="#5d2b90" HeaderTitle="Razor"
+                     Content="PFN0aWNreVRpdGxlQmFyPgogIDxTdGlja3lUaXRsZUJhckxlZnQ+CiAgICA8QnV0dG9uIENsYXNzPSJtZW51LWljb24iIGRhdGEtb3Blbj0ib2ZmQ2FudmFzTGVmdCI+PC9CdXR0b24+CiAgICBCYWx6b3IgRm91bmRhdGlvbgogIDwvU3RpY2t5VGl0bGVCYXJMZWZ0PgogIDxTdGlja3lUaXRsZUJhclJpZ2h0PgogICAgPEJ1dHRvbiBDbGFzcz0ibWVudS1pY29uIiBkYXRhLW9wZW49Im9mZkNhbnZhc1JpZ2h0Ij48L0J1dHRvbj4KICA8L1N0aWNreVRpdGxlQmFyUmlnaHQ+CjwvU3RpY2t5VGl0bGVCYXI+CjxPZmZDYW52YXMgUG9zaXRpb249InBvc2l0aW9uLWxlZnQiIElkPSJvZmZDYW52YXNMZWZ0IiBzdHlsZT0iYmFja2dyb3VuZC1jb2xvcjogIzExMTsiPgogICAgPHVsIHN0eWxlPSJjb2xvcjojZmZmZmZmIj4KICAgICAgICA8bGk+QWJvdXQ8L2xpPgogICAgICAgIDxsaT5TZXJ2aWNlczwvbGk+CiAgICAgICAgPGxpPkNsaWVudHM8L2xpPgogICAgICAgIDxsaT5Db250YWN0PC9saT4KICAgIDwvdWw+CjwvT2ZmQ2FudmFzPgo8T2ZmQ2FudmFzIFBvc2l0aW9uPSJwb3NpdGlvbi1sZWZ0IiBJZD0ib2ZmQ2FudmFzUmlnaHQiPgogICAgPHVsPgogICAgICAgIDxsaT5BYm91dDwvbGk+CiAgICAgICAgPGxpPlNlcnZpY2VzPC9saT4KICAgICAgICA8bGk+Q2xpZW50czwvbGk+CiAgICAgICAgPGxpPkNvbnRhY3Q8L2xpPgogICAgPC91bD4KPC9PZmZDYW52YXM+" />
+
+        <OpenCodeDev.Blazor.Foundation.Components.Plugins.Sticky.StickyTitleBar>
+            <OpenCodeDev.Blazor.Foundation.Components.Plugins.Sticky.StickyTitleBarLeft>
+                <Button Class="menu-icon" data-open="offCanvasLeft"></Button>
```

Comment:
??? icon ?? why not use MDI span for icon button makes it very ugly

Reply:
```suggestion
                <span class="mdi mdi-menu" data-open="offCanvasLeft"></span>
```